ocs_ci/deployment/helpers/external_cluster_helpers.py

```diff
@@ -137,6 +143,25 @@ def upload_exporter_script(self):
         upload_file(self.host, script_path, script_path, self.user, self.password)
         return script_path
 
+    def upload_rgw_cert_ca(self):
+        """
+        Upload RGW Cert CA to RHCS cluster
+
+        Returns:
+            str: absolute path to the CA Cert
+
+        """
+        rgw_cert_ca_path = get_and_apply_rgw_cert_ca()
+        remote_rgw_cert_ca_path = "/tmp/rgw-cert-ca.pem"
```

Comment:
Not sure how this tmp file is used, if it's one time only. If we will need it again, later on, and the machine will reboot, we might loose this file. Is that OK?

Reply:
It is used one time only - when executed the external cluster exporter script, so it should be ok.